evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -978,6 +981,7 @@ def test_score_with_objective_that_requires_predict_proba(mock_predict, dummy_re
         clf = dummy_regression_pipeline_class(parameters={})
         clf.fit(X, y)
         clf.score(X, y, ['precision', 'auc'])
+    # Why don't we use pytest.raises here?
```

Comment:
@freddyaboulton delete this comment?

As to the question: I don't think there's any particular reason! `pytest.raises` would work fine here. I guess we're checking for multiple substrings in the error message, but I think there's a way to access the exception with `pytest.raises`...

Reply:
@angela97lin oooh, great detective work! So it was for codecov, interesting. We've relaxed the limits for codecov a bit since then, so this may work now 🤷‍♂️

Reply:
Of course I was confused about my own code 😅  I will delete the comment and keep as-is. Thanks for digging into this @angela97lin !